### ISSUE TITLE
No models to download from the app

### ISSUE THREAD

**Issue by PumpkinSheee** [tryreplay/replay-issues#41]
dont know if its an issue or not, but i cant download any models from the replay app, they just dont appear
![image](https://github.com/tryreplay/replay-issues/assets/109385410/e35fb50c-5795-4e0d-abe7-3583362b3a71)

**Comment by jonluca** [comment]
Yes you can download models from weights.gg, the ones in the app were kind of broken

**Comment by PumpkinSheee** [comment]
damn
